src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Main {
```

Comment:
There should not be an empty line at the start of a class or method implementation. Please remove the empty line after the package declaration to follow best practices.